packages/react-native/ReactCommon/react/renderer/components/textinput/platform/android/react/renderer/components/androidtextinput/AndroidTextInputShadowNode.cpp

```diff
@@ -45,6 +45,15 @@ AndroidTextInputShadowNode::AndroidTextInputShadowNode(
   }
 }
 
+void AndroidTextInputShadowNode::initialize() {
+  if (yogaNode_.style().display() == yoga::Display::Contents) {
+    yogaNode_.style().setDisplay(yoga::Display::None);
+  }
```

Comment:
Could you link to spec about why we do this? Though, I'm not sure if it is safe to mutate Yoga node at this point. Maybe makes sense to have some sort of trait to control this which is read when we are transferring props to Yoga node.

Reply:
https://drafts.csswg.org/css-display/#unbox-html

I assumed that `TextInput` would correspond to `input`.